_rules/SC4-1-2-aria-attribute-has-valid-value.md

```diff
@@ -0,0 +1,255 @@
+---
+name: ARIA attribute has valid value
+test_type: atomic
+
+description: |
+   This rule checks that all ARIA 1.1 attributes (`aria-*`) have valid values
+   
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any [non-empty](#non-empty) attribute that is specified on an HTML or SVG element, where the attribute name starts with `aria-` and can be found in [WAI-ARIA 1.1, Definitions of States and Properties](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def).
```

Comment:
We don't need to specify that the attribute name starts with `aria-`. All names in the WAI-ARIA definition do, so it is redundant.
On the other hand, I understand if you wish to keep it, since it will help to optimize the applicability decision process.

Reply:
Thinking of it, I tend to agree with Carlos. Is it not sufficient to say "where attribute is listed in ..."?

Reply:
I think the idea was to exclude role="", that is also an attribute listed in WAI-ARIA 1.1...